atlasdb-config/src/test/java/com/palantir/atlasdb/http/FailoverFeignTargetTest.java

```diff
@@ -176,15 +172,10 @@ public void exceptionsWithRetryAfterBacksOffAfterQueryingAllNodesInTheCluster()
             spiedTarget.continueOrPropagate(EXCEPTION_WITH_RETRY_AFTER);
         }
 
-        ArgumentCaptor<Long> argument = ArgumentCaptor.forClass(Long.class);
-        verify(spiedTarget, times(1)).pauseForBackoff(any(), argument.capture());
-
-        List<Long> arguments = argument.getAllValues();
-
         long bottom = FailoverFeignTarget.BACKOFF_BEFORE_ROUND_ROBIN_RETRY_MILLIS / 2;
         long cap = (FailoverFeignTarget.BACKOFF_BEFORE_ROUND_ROBIN_RETRY_MILLIS * 3) / 2;
```

Comment:
You can extract these constants. Also would rename them to something more descriptive like `LOWER_BOUND` and `UPPER_BOUND`